### PR TITLE
Added Business of Fashion

### PR DESCRIPTION
**Pull request by AshNL**
In response to a Site Request which has since been closed for not following the template. I had already completed the code but was unable to create PR's.

This is my first ever contribution to a project and the first time manipulating the DOM using JS, so I'm open to feedback and there is probably a better way to select the elements stored in `paywallBox` & `professionalMemberAd`.

### CODE REVIEW

File: src/js/contentScript.js [hunk 1]
```diff
@@ -338,6 +338,22 @@ if (matchDomain('estadao.com.br')) {
 } else if (matchDomain('nzz.ch')) {
   const paywall = document.querySelector('.dynamic-regwall');
   removeDOMElement(paywall);
+} else if (matchDomain('businessoffashion.com')) {
+  const paywall = document.querySelector('.article-locked');
+  const paywallOverlay = document.querySelector('.article-locked-overlay');
+  // paywallBox & professionalMemberAd need cleaner selectors. Help appreciated.
+  const paywallBox = document.querySelector("#heading > div.col-md-12.col-xs-24.col-sm-offset-1.col-md-offset-0.col-sm-22.print-full-width.paywalled-content > div > div > wall-handler-subscription-wrapped > div > div.row.padding-horizontal-xs-2.margin-bottom-xs-5.margin-bottom-md-0 > div");
```
Comment (iamadamdev) [inline src/js/contentScript.js:345]: It's not always reliable to remove it right away, so we can add a small delay before removing it, something like this:
setTimeout(function () {
  const bottomContainer = document.querySelector(".smart-slot-position-global-screen-bottom__container");
  removeDOMElement(bottomContainer);
}, 500); // Delay (in milliseconds)

Comment (AshNL) [reply src/js/contentScript.js]: Makes sense. I've added a 500ms delay.

Comment (iamadamdev) [reply src/js/contentScript.js]: I meant to also remove paywallBox and professionalMemberAd and replace with:
```
const bottomContainer = document.querySelector(".smart-slot-position-global-screen-bottom__container");
removeDOMElement(bottomContainer);
```

does that work for you?

Comment (AshNL) [reply src/js/contentScript.js]: Ah my apologies. I thought `bottomContainer` was a random example to clarify the `setTimeout` function. Removing that element does indeed do the job. I'll commit to my master.

**Comment by iamadamdev** [comment]
Closed due to inactivity.

**Pull request closed without merging**